Helper/Utils.php

```diff
@@ -0,0 +1,299 @@
+<?php
+
+
+namespace Midtrans\Snap\Helper;
+
+
+class Utils
+{
+
+    /**
+     * Replace string for items name
+     *
+     * @param $str
+     * @return string|string[]|null
+     */
+    public function repString($str)
+    {
+        return preg_replace("/[^a-zA-Z0-9]+/", " ", $str);
+    }
+
+    /**
+     * Sanitize for item name
+     *
+     * @param $s
+     * @return false|string
+     */
+    public function getName($s)
```

Comment:
hmm function name seems to be not so descriptive, I guess the purpose of this function is to "trim" excess string length, so that max length is 20 char.

Also I see it usually used together with previous function:
```
$this->repString($this->getName($newItem->getName()))
```

Maybe merge the 2 function, so that when calling from Controller/Model you just call:
```
$this->sanitizeItemName($newItem->getName())
```

Then internally the `sanitizeItemName` utils function can call `trimLength` & `sanitizeString` internal function.

Reply:
Done, refactored to one function sanitizeItemName